cmds/admin_server/server/server.go

```diff
@@ -77,14 +78,17 @@ func (r *RouteHandler) status(c *gin.Context) {
 
 // addLog inserts a new log entry inside the database
 func (r *RouteHandler) addLog(c *gin.Context) {
+	logger := r.logger
+
 	var log Log
 	if err := c.Bind(&log); err != nil {
 		c.JSON(http.StatusBadRequest, gin.H{"status": "err", "msg": "badly formatted log"})
-		fmt.Fprintf(os.Stderr, "Err while binding request body %v", err)
+		logger.Errorf("Err while binding request body %v", err)
```

Comment:
shorter if you rename the field; no need for the extra var

```suggestion
		r.log.Errorf("Err while binding request body %v", err)
```